app/controllers/idv/agreement_controller.rb

```diff
@@ -36,7 +36,6 @@ def update
       )
 
       if result.success?
-        idv_session.idv_consent_given = true
```

Comment:
I think we need to keep these around for 1 more release cycle. Old instances will still be inspecting `idv_session.idv_consent_given`. We need to stop reading from these everywhere before we can stop writing it.

Reply:
Oh, good catch! [The first PR](https://github.com/18F/identity-idp/pull/11164/files) started setting the timestamp, allowing this to begin reading it immediately, but I overlooked the inverse which will occur in the 50/50 state.